src/test/java/org/broadinstitute/hellbender/tools/coveragemodel/Nd4jUnitTest.java

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.nd4j.linalg.api.buffer.DataBuffer;
+import org.nd4j.linalg.factory.Nd4j;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * This test is to ensure that the DType is correctly set to Double
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public class Nd4jUnitTest extends BaseTest {
+    @Test
+    public void testDType() {
+        Assert.assertTrue(Nd4j.dataType().equals(DataBuffer.Type.DOUBLE), "Data type for Nd4j must be set to" +
```

Comment:
Nd4j folks work with half- and single-precision more often than double-precision as they mostly care about GPUs. It has happened at least twice that they broke `Nd4j.setDataType` and default dtype-setting via system properties! while `CoverageModelEMWorkspace` performs its own data type test, I thought including it in the test suite is a good idea to catch this issue early on in the future when we upgrade to newer nd4j builds.